### DISCUSSION TITLE
Checkbox for boolean values

### DISCUSSION THREAD

**Discussion by vanillajonathan** [minhhungit/ConsoleTableExt#13] [Ideas]
A checkbox or some sort of Unicode symbol could be used represent boolean values.

- ☐ - U-26110 ([compart](https://www.compart.com/en/unicode/U+2610))
- ☑ - U-26111 ([compart](https://www.compart.com/en/unicode/U+2611))
- ☒ - U-26112 ([compart](https://www.compart.com/en/unicode/U+2612))

https://en.wikipedia.org/wiki/X_mark

**Comment by minhhungit** [comment]
Answered here https://github.com/minhhungit/ConsoleTableExt/discussions/14#discussioncomment-352757
Will use Formatter

**Reply by vanillajonathan** [reply]
Sounds great! Using a formatter would be the most flexible way so the user can define which Unicode symbol to use.
